course/lesson-2-your-first-error/content-UdOCQiGj.tres

```diff
@@ -6,9 +6,9 @@
 script = ExtResource( 1 )
 title = ""
 type = 0
-text = "Errors are messages designed on purpose by fellow programmers who came before you. They anticipated you might get into specific issues.
+text = "Errors are messages designed on purpose by fellow programmers who came before you. They anticipated you might have specific issues.
 
-You shouldn't think of an error as a failure: it is a mentor helping you from the past. Importantly, errors won't break your computer. At least not with GDScript: it's a pretty safe language."
+You shouldn't think of an errors as a failures. They are like mentors helping you from the past. Importantly, errors won't break your computer. At least not with GDScript because it's a pretty safe language."
```

Comment:
```suggestion
You shouldn't think of errors as failures. They are like mentors helping you from the past. Importantly, errors won't break your computer. At least not with GDScript because it's a pretty safe language."
```